packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -406,6 +416,30 @@ - (void)updatePlayingState {
   _displayLink.running = _isPlaying || self.waitingForFrame;
 }
 
+- (void)updateRate {
```

Comment:
This needs a declaration comment, per the style guide.

Reply:
Like for example? I cannot see any such comment above any other such function except maybe `disposeSansEventChannel`. What could be put here, something like "Sets the rate of the player."?

Reply:
> I cannot see any such comment above any other such function except maybe `disposeSansEventChannel`.

Unfortunately the reviews for a lot of our older plugin code didn't enforce [the style guide](https://google.github.io/styleguide/objcguide.html#declaration-comments). You're certainly welcome to fix other methods if you would like to, but since you didn't add those methods they weren't part of my review.

> What could be put here, something like "Sets the rate of the player."?

A meaningful description of what the method does. For instance: "Synchronizes the `player`'s playback rate with `playbackSpeed`, constrained by the playback rate capabilities of the player."